homework/App.js

```diff
@@ -18,17 +18,42 @@ class App {
 
     const root = document.getElementById('root');
 
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
+    const container = Util.createAndAppend('div', root, {
```

Comment:
There are still some unneeded comments above this declaration. Could you please remove them? 🙂

Reply:
It's done! 👍 :)
Thanks for your feedback!